cypress/e2e/gui/detalheCliente.cy.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress" />
+
+describe('Detalhes do cliente', () => {
+    it('Campo entrada de texto deve estar desabilitado', () => {
+        cy.visit('http://localhost:3000/')
```

Comment:
Que tal mover o `cy.visit()` para o _hook_ `beforeEach`?

Dessa forma, tal comando não precisaria se repetir para cada bloco `it`.

Além disso, caso a `baseUrl` esteja definida no arquivo `cypress.config.js`, tal comando poderia ser só `cy.visit('/')`.

Reply:
obrigada pela dica